testnet/stacks-node/src/neon_node.rs

```diff
@@ -885,22 +928,26 @@ fn spawn_miner_relayer(
                     }
                     bump_processed_counter(&blocks_processed);
                 }
-                RelayerDirective::BroadcastMicroblock(
-                    parent_consensus_hash,
-                    parent_block_hash,
-                    microblock,
-                ) => {
-                    let microblock_hash = microblock.block_hash();
-                    if let Err(e) = relayer.broadcast_microblock(
-                        &parent_consensus_hash,
-                        &parent_block_hash,
-                        microblock,
-                    ) {
-                        error!(
-                            "Failure trying to broadcast microblock {}: {}",
-                            microblock_hash, e
-                        );
+                RelayerDirective::RunMicroblockTenure => {
```

Comment:
Can `RunMicroblockTenure` directives be coalesced? As implemented, I think we'd end up with many `RunMicroblockTenure` directives in the channel at the same time, leading the mining thread to try mining a microblock tenure continuously, when the mempool is unlikely to have changed (and these 0-tx attempts to mine a microblock are not cheap).

Reply:
Sure -- I can make the relayer aware of the expected time between microblocks and ignore these directives if the time since the last microblock was mined is earlier than the deadline to start mining the next one.

Reply:
Fixed in 1d977c1